src/rust/inetstack/protocols/layer4/tcp/tests/simulator.rs

```diff
@@ -1,11 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+// Todo: Move this component to the `network-simulator` crate.
```

Comment:
can you make this TODO: so it will be flagged by our vscode extension?

Reply:
Done